docs/overview/index.rst

```diff
@@ -27,3 +27,4 @@ Overview
     renderers/index
     new_widget
     vg_lite_tvg
+    tools
```

Comment:
I suggest creating a `debugging` folder instead where we can add
- gdb
- profiler
- logging
- VGLite simulator